### PR TITLE
Fix for issue https://wso2.org/jira/browse/APIMANAGER-5624

### PR DESCRIPTION
**Pull request by tharikaGitHub**

### CODE REVIEW

File: components/web/apps/org.wso2.carbon.apimgt.editor.app/src/main/fragments/action-menu-api-details/action-menu-api-details.hbs [hunk 1]
```diff
@@ -10,7 +10,8 @@
             </a>
         </li>
         <li>
-            <a href="{{env.contextPath}}/version?name=cha123&amp;version=1.0.0&amp;provider=admin" title="Create New Version">
+        {{!--{{env.contextPath}}/version?name=cha123&amp;version=1.0.0&amp;provider=admin--}}
+            <a href="#" onclick="return alert('This function is not supported yet')" title="Create New Version">
```
Comment (dakshika) [inline components/web/apps/org.wso2.carbon.apimgt.editor.app/src/main/fragments/action-menu-api-details/action-menu-api-details.hbs]: Adding default js alert isn't recommend to use in wso2 products. use standard lib to show notifications and errors

File: components/web/apps/org.wso2.carbon.apimgt.editor.app/src/main/fragments/action-menu-api-details/action-menu-api-details.hbs [hunk 2]
```diff
@@ -19,10 +20,11 @@
             </a>
         </li>
         <li>
-            <a href="{{env.contextPath}}/version?name=cha123&amp;version=1.0.0&amp;provider=admin" title="Create New Version">
+        {{!--{{env.contextPath}}/version?name=cha123&amp;version=1.0.0&amp;provider=admin--}}
+            <a href="#" onclick="return alert('This function is not supported yet')" title="Delete">
```
Comment (dakshika) [inline components/web/apps/org.wso2.carbon.apimgt.editor.app/src/main/fragments/action-menu-api-details/action-menu-api-details.hbs]: Adding default js alert isn't recommend to use in wso2 products. use standard lib to show notifications and errors

File: components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/pages/tierPermissions/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+function onGet(env) {
+    return {};
+}
```
Comment (sumedhassk) [inline components/web/apps/org.wso2.carbon.apimgt.publisher/src/main/pages/tierPermissions/index.js]: Better to have newlines at the end of the files.

**Comment by tharikaGitHub** [comment]
Thanks for the review @dakshika and @sumedhassk. Since we have discontinued UUF in favour of React.js, the files action-menu-api-details.hbs and tierPermissions/index.js are no longer available in this repository.

**Pull request merged**